src/interactive/commands.sml

```diff
@@ -66,14 +66,15 @@ struct
             INL s => raise Fail s
           | INR sign =>
             let
-              val elab = RefineElab.transport o ValidationElab.transport o BindSignatureElab.transport
-              val sign' = elab sign
+              val elab = ValidationElab.transport o BindSignatureElab.transport
+              val sign' = RefineElab.transport (AbtSignature.Telescope.append (oldSign, elab sign))
```

Comment:
@ak3n If it seems like we need to convert symbol telescopes to string telescopes, I think we've taken a wrong turn... Can you think of any other way to do what you need?